index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio Website</title>
+</head>
+<body>
+    <header>
+        <div class="logo">
+          <a>My Logo</a>
+        </div>
+        <div class="nav-bar">
+            <img src="image/Vector.png" alt="Nav Bar" class="navImg1">
+            <img src="image/Vector.png" alt="Nav Bar" class="navImg2">
+            <img src="image/Vector.png" alt="Nav Bar" class="navImg3">
+
+        </div>
+    </header>
+    <main>
+       <section>
+           <div class="Backgroud">
+  
+    
+           </div>
```

Comment:
- [Optional] I see you made use of a CSS background image to set the background image, I advise you can ref the `<main>` tag with a class in your CSS file and give that the background-image property as it holds all element in that section.